handbook/engineering/frontend-platform/index.md

```diff
@@ -0,0 +1,45 @@
+<img src="./logo.svg" alt="Sourcegraph Web Team Logo" style="width: 35%; float: right; margin-left: 1rem">
```

Comment:
How about we move the `frontend-platform` (and code insights folder) to a subfolder of the `web` folder, mirroring the org structure?

Reply:
Good idea, will do. I considered it but decided to follow the pattern that Code Insights had already set :-)

Reply:
Done.